src/index.html

```diff
@@ -1,14 +1,57 @@
 <!doctype html>
 <html lang="en">
+  
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="moyo_1">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
```

Comment:
Add empty lines between multiline sibling blocks of HTML
<img width="1230" alt="Screenshot 2023-11-09 at 17 39 25" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/2857089c-0378-4f83-8035-17cf05666d65">